mortar/src/main/java/mortar/MortarScope.java

```diff
@@ -30,8 +30,20 @@
   public static final String DIVIDER = ">>>";
   public static final String SERVICE_NAME = MortarScope.class.getName();
 
+  /**
+   * Retrieves a MortarScope from the given context. If none is found, retrieves a MortarScope from
+   * the application context.
+   *
+   * @return null if no scope is found in either the given context or the application context
+   */
   public static MortarScope getScope(Context context) {
-    return (MortarScope) context.getSystemService(MortarScope.SERVICE_NAME);
+    //noinspection ResourceType
+    Object scope = context.getSystemService(SERVICE_NAME);
+    if (scope == null) {
+      //noinspection ResourceType
+      scope = context.getApplicationContext().getSystemService(SERVICE_NAME);
```

Comment:
Figure it out with @loganj . What's going on is that the root mBase is ContextImpl. Early in the lifecycle, it doesn't have access to system services. However, it has access to the application context by going through its package info. And that's why this works.